src/lib/logged-in/debt-tracker/people/person-delete/person-delete.component.ts

```diff
@@ -0,0 +1,39 @@
+import {Component, computed, inject, Input, OnInit, Signal, WritableSignal} from "@angular/core"
+import { ContactService } from "api/contacts/contact.service"
+import {ReactiveFormsModule} from '@angular/forms'
+import {Dialog} from 'primeng/dialog'
+import {Button} from 'primeng/button'
+import {PrimeTemplate} from 'primeng/api'
+
+
+@Component({
+  selector: 'dbt-person-delete',
+  templateUrl: './person-delete.component.html',
+  standalone: true,
+  imports: [
+    Dialog,
+    ReactiveFormsModule,
+    Button,
+    PrimeTemplate,
+  ]
+})
+export class PersonDeleteComponent implements OnInit {
+  ngOnInit(): void {}
+  private readonly contactService = inject(ContactService)
+  @Input() personId!: WritableSignal<string>
```

Comment:
Use signal inputs and outputs

```suggestion
  readonly personId = input<string>()
```